Command/GenerateEntitiesDoctrineCommand.php

```diff
@@ -72,6 +72,9 @@ class is supposed to extend which. You have to adjust the entity
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        trigger_error('The doctrine:generate:entity command has been deprecated.', E_USER_DEPRECATED);
+        $output->writeln(' <comment>NOTE:</comment> The <info>doctrine:generate:entities</info> command has been deprecated. Use <info>make:entity --regenerate</info> from MakerBundle instead.');
```

Comment:
I'd likely suggest pointing out the bad practice and point it at the docs instead. We (doctrine project) do not endorse MakerBundle in any way, as it just perpetuates an already too widespread misuse of mappings.

Reply:
I'd suggest changing the language to something like this:

    $output->writeln(' <comment>NOTE:</comment> The <info>doctrine:generate:entities</info> command has been deprecated. To read more about the differences between anemic and rich models go here <link to our docs>. If you wish to generate your entities, use <info>make:entity --regenerate</info> from MakerBundle instead.');